api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/MessageQueueDAO.java

```diff
@@ -29,21 +30,26 @@
 
     @Override
     public void sendMessage(MessagePayload messagePayload, MessageType messageType) {
```

Comment:
Would it be worth renaming this to `sendMessageImmediately` or something similar so the difference is clear?

Reply:
I think the lack of any period or date is sufficient.